### ISSUE TITLE
Web interface never appearing

### ISSUE THREAD

**Issue by OlMi1** [Snipa22/nodejs-pool#493]
I ran everything in the readme just like it is stated there but I am stuck at step 6. Connecting to /admin.html & :8080 (without / at the end or at any subfolder, just IP:8080) results in a "connection refused" even though I started the poolui stuff correctly (running on localhost:8080). What should I do now? Opening the port does not help. (sudo ufw allow 8080)

**Comment by S1700** [comment]
Hold on have you done this step?

https://github.com/Snipa22/nodejs-pool#manual-sql-configuration

**Comment by OlMi1** [comment]
Didn't even get there. In the end of the deploy.bash the API should start so this is not necessarry, port 8080 appears when manually starting the poolui (and has nothing to do with the API). 
Currently working on my own fork of it with improved docs and already edited a lot but I'm not really understanding what 
en`v PATH=$PATH:`pwd`/.nvm/versions/node/v8.9.3/bin node sql_sync.js`
wants to do since it just errors... I'm completely lost on this shit repo. I am getting a connection refused on 8080 too and the readme here seems to be absolute shit. 
It would be great to exchange info about the issue if one of us finds any.

**Comment by S1700** [comment]
yh ill try solving this, good luck with your own fork

**Comment by OlMi1** [comment]
Thank you. My fork is basically just meant to improve the readme and is exactly same to this pool so if you find something or I do it would apply to both out issues.

**Comment by OlMi1** [comment]
> We have the same issue and we are getting `Connection Refused` on /admin.html and on port 8001

Okay so I think I found how we can check. Run `pm2 list` and the namespance `API` should appear, I think?

**Comment by S1700** [comment]
When I run `pm2 list` I do have the API module running

**Comment by OlMi1** [comment]
Is it displayed in green ('running', 'ok' or sth like that)? And where do you host the pool?

**Comment by S1700** [comment]
yup, it's green and so is everything else. What do you mean by where do I host it. like the provider?

**Comment by helly2** [comment]
> I ran everything in the readme just like it is stated there but I am stuck at step 6. Connecting to /admin.html & :8080 (without / at the end or at any subfolder, just IP:8080) results in a "connection refused" even though I started the poolui stuff correctly (running on localhost:8080). What should I do now? Opening the port does not help. (sudo ufw allow 8080)
> 
> a curl to localhost:8080 works.
> 
> My firewall config is:
> 
> ```
> To                         Action      From
> --                         ------      ----
> 8080                       ALLOW IN    Anywhere
> 8080/tcp                   ALLOW IN    Anywhere
> 8080 (v6)                  ALLOW IN    Anywhere (v6)
> 8080/tcp (v6)              ALLOW IN    Anywhere (v6)
> ```

im looking at your fork but cant write issues there, is there any way to notify you
